package.json

```diff
@@ -101,16 +103,21 @@
     "istanbul": "^0.4.5",
     "jasmine": "^2.6.0",
     "jasmine-spec-reporter": "^3.2.0",
+    "karma-jasmine-html-reporter": "^0.2.2",
     "license-checker": "^20.1.0",
     "minimatch": "^3.0.4",
     "minimist": "^1.2.0",
+    "npm-registry-client": "^8.6.0",
+    "pidtree": "^0.3.0",
```

Comment:
Shouldn't these dependencies be spread out over the packages that use them instead? 

Having them here is odd. Why are these here but not others? The choice of packages to include here seems like a side effect of how yarn is hoisting packages.

Reply:
Only dependencies used by the development scripts and build system should be included here

Reply:
But these seem arbitrary compared to the variety of other dependencies used by the packages that are not added here.

A single directory per Bazel package I think would be preferable over a single directory per Bazel workspace.  Each package may have varying and conflicting npm dependencies.

> yarn workspaces requires the modules to be layed out in many directories.

it's not really yarn workspaces but rather the packages themselves and their dependencies that require it.  Otherwise the packages are not receiving the dependencies they specify.

Reply:
This will be addressed by the upcoming work in https://angular-team.atlassian.net/browse/TOOL-463